services-js/registry-certs/client/birth/ReviewRequestPage.tsx

```diff
@@ -201,14 +176,51 @@ const CERTIFICATE_ROW_STYLE = css({
 
   display: 'flex',
   alignItems: 'center',
+
+  '> div:first-of-type': {
+    flexBasis: '25%',
+  },
+});
+
+const BUTTONS_CONTAINER_STYLING = css({
```

Comment:
Can this all be accomplished just with our grid classes?

Reply:
(I just realized that the change to the `QuestionsComponent` flow control buttons demonstrates the behavior I’m trying to avoid...)

I wanted to keep the element source order in line with the visual order on the desktop so tabbing behavior makes sense to a keyboard user. However, it _also_ makes sense for a mobile user to have the “next” button come first. Because flexbox’s `direction` and `order` only affect visual presentation, I took advantage of that method here.

I’d like to go back and use this method in `QuestionsComponent` as well, so let’s chat. 😀